red/red_install_raspbian.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Installing on Raspbian
+sidebar: red_sidebar
+permalink: /red_install_raspbian/
+last_updated: June 13, 2016
+toc: true
+---
+
+The installation on Raspbian differs quite a bit from the a traditional Linux installation.
+Due the fact that Raspbian does not natively support Python 3.5 (and above),
+and that Raspbian does not support Ffmpeg. In this guide there is a easy fix for all these problems.
+
+# Installing with a script (Easiest)
+For ease of access and installing.
+You can use the following ``.sh`` script to install all the pre-requirements.
+
+### Downloading the ``.sh`` to your Raspbian
+To acquire and run the script. you must transfer it to your server with sftp or with ``wget``,
+for the sake of simplicity we will use ``wget``
+
+```
+wget https://twentysix26.github.io/Red-Docs/files/pi.sh
+sudo chmod +x pi.sh
+```
+
+### Running the script
+Next we will run the script.
+This script will install pre-requirements to run your very own Red bot on your Pi.
+
+{{site.data.alerts.warning}}The installation of the pre-requirements can vary from at least 30 minutes to 90 minutes, depending on the model of Pi you own.{{site.data.alerts.end}}
+
+```
+./pi.sh
+```
+
+Follow the instructions that the script gives you.
+Don't worry, you only need to enter one thing and you're done.
+
+In case you don't know your number of CPU cores, see the sheet below.
+
+| Pi 1 	| Pi 1+ 	| Pi 1B 	| Pi 1B+ 	| Pi 2B 	| Pi 3 	|
+|------	|-------	|-------	|--------	|-------	|------	|
+| 1    	| 1     	| 1     	| 1      	| 4     	| 4    	|
+
+### Cloning and preparing Red
+Once the script says the following you can start cloning Red and installing it.
+From here you need to generate your new bot account.
```

Comment:
can just remove this line `From here you need to generate your new bot account.`